src/mocks/handlers/users.ts

```diff
@@ -0,0 +1,13 @@
+import { HttpResponse } from 'msw'
+import { http } from '@/mocks/http'
+import type { components } from '@/api/schema'
+
+type UserResponse = components['schemas']['UserResponse']
+
+const myID = import.meta.env.MY_TRAQ_ID || 'kitsne'
```

Comment:
`myId`が変更できるようにしてる意図は何ですか？
特に理由がなければ固定で良いかなと思います

デフォルト値ですが、卒業後のアイコン表示等考えれば`'traP'`が丸いと思います

'myID`はtypoだと思うので修正お願いします
```suggestion
const myId = 'traP'
```

Reply:
本番環境に近い見た目を再現して作業をする方がよいと考えたから、という以上の理由はないので、仰る通り `traP` でよいですね

卒業メンバーのアイコンは traQ の API には残っています（trap.jp の過去のブログの著者のアイコンは正常に表示されている）。将来的な話ですが、部屋のメンバーなど多数のユーザーを返すモックを考えるときにはどんなデータを使えばよいのか、それをこの公開リポジトリにハードコードしてよいのか、などについては少し検討が必要かも知れません